lib/smart_answer/calculators/check_benefits_support_calculator.rb

```diff
@@ -59,7 +59,7 @@ def eligible_for_housing_benefit?
 
     def eligible_for_tax_free_childcare?
       eligible_child_ages = %w[1_or_under 2 3_to_4 5_to_11]
-      @are_you_working == "yes" &&
+      @are_you_working == "yes_over_16_hours_per_week" || "yes_under_16_hours_per_week" &&
```

Comment:
Could we make this `@are_you_working != "no"`?
I think we would need to wrap this as `(@are_you_working == "yes_over_16_hours_per_week" || @are_you_working == 
 "yes_under_16_hours_per_week") && ...` otherwise to prevent it evaluating to `yes_under_16_hours_per_week` if it's not set to `yes_over_16_hours_per_week`

Reply:
thanks! I have fixed up @edwardkerry